plugins/package-managers/python/src/main/kotlin/Pip.kt

```diff
@@ -102,6 +116,8 @@ class Pip(
     ): PythonInspector.Result {
         val operatingSystem = operatingSystemOption ?: OPTION_OPERATING_SYSTEM_DEFAULT
         val pythonVersion = pythonVersionOption ?: detectPythonVersion() ?: OPTION_PYTHON_VERSION_DEFAULT
+        val analyzeSetupPyInsecurely =
+            analyzeSetupPyInsecurelyOption?.toBoolean() ?: OPTION_ANALYZE_SETUP_PY_INSECURELY_DEFAULT
```

Comment:
Nit: We prefer to wrap lines as late as possible in this case, so better write

        val analyzeSetupPyInsecurely = analyzeSetupPyInsecurelyOption?.toBoolean()
            ?: OPTION_ANALYZE_SETUP_PY_INSECURELY_DEFAULT

Reply:
Also, when using `toBooleanStrictOrNull()` above, this should become `toBooleanStrict()`.